augly/image/utils.py

```diff
@@ -279,3 +280,23 @@ def compute_stripe_mask(
     binary_mask = softmax_mask > (math.cos(math.pi * line_width) + 1) / 2
 
     return binary_mask
+
+
+def distort(
+        image: Image.Image,
+        method: str,
+        distortion_args: Tuple,
+) -> Image.Image:
+    """
+    Distorts the image with a specified type of distortion. This function wraps `Wand` package
+    `wand.image.Image.distort()` method to apply distortions. This function is a helper function
+    to apply lens distortions on the image, and it is not meant to be used for methods explicitly
+    written in AugLy. To see full set of distortion methods, see
+    https://docs.wand-py.org/en/0.5.3/wand/image.html#wand.image.DISTORTION_METHODS
+    """
+    image = np.array(image)
+    wimage = wImage.from_array(image)
```

Comment:
```suggestion
    npimage = np.array(image)
    wimage = wImage.from_array(npimage)
```

nit: Our internal linter is complaining that image is declared to have type `Image.Image` but is used as type `np.ndarray`. Renaming the var when converting it to np will fix this.